wgengine/router/router_userspace_bsd.go

```diff
@@ -153,17 +153,23 @@ func (r *userspaceBSDRouter) Set(cfg *Config) error {
 }
 
 func (r *userspaceBSDRouter) Close() error {
-	cleanup(r.logf, r.tunname)
+	if err := downDNS(interfaceName); err != nil {
+		logf("dns down: %v", err)
```

Comment:
does this compile? where is logf?

Reply:
Let me quickly test this on an actual FreeBSD machine; I just adapted it from #615.